docs/architecture/decisions/0002-docker-for-development.md

```diff
@@ -0,0 +1,25 @@
+# 2. Docker for development
+
+Date: 2022-08-09
+
+## Status
+
+Accepted
+
+## Context
+
+The [Federal Audit Clearinghouse](https://github.com/GSA-TTS/FAC) project is a similar project to ours, from which we are taking many examples for reuse. FAC uses Docker. Docker is widely accepted at TTS currently and in the development community at large.
```

Comment:
Might add that Docker works well with cloud.gov and is very portable to other cloud solutions.

Reply:
Added, ty.